src/main/java/io/fabric8/vertx/maven/plugin/mojos/RunMojo.java

```diff
@@ -33,8 +36,111 @@
     requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class RunMojo extends AbstractRunMojo {
 
+    /**
```

Comment:
one more thing i felt was making user matching `vert.x` options helps them to understand what they are doing and what they have added.